dist/src/main/java/io/camunda/zeebe/broker/TopologyServices.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.broker;
+
+import io.atomix.cluster.AtomixCluster;
+import io.atomix.cluster.ClusterMembershipService;
+import io.camunda.zeebe.gateway.impl.broker.cluster.BrokerTopologyManager;
+import io.camunda.zeebe.gateway.impl.broker.cluster.BrokerTopologyManagerImpl;
+import io.camunda.zeebe.scheduler.ActorScheduler;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.stereotype.Component;
+
+@Component
+public class TopologyServices {
+  private final ActorScheduler scheduler;
+
+  private final ClusterMembershipService clusterMembershipService;
+
+  @Autowired
+  public TopologyServices(final ActorScheduler scheduler, final AtomixCluster atomixCluster) {
+    this.scheduler = scheduler;
+    clusterMembershipService = atomixCluster.getMembershipService();
+  }
+
+  @Bean
+  BrokerTopologyManager brokerTopologyManager() {
+    final var brokerTopologyManager =
+        new BrokerTopologyManagerImpl(clusterMembershipService::getMembers);
+    scheduler.submitActor(brokerTopologyManager).join();
+    clusterMembershipService.addListener(brokerTopologyManager);
+    // TODO: register with gossip
```

Comment:
:question: This is registered later in the broker, right? So this TODO is not valid?

Reply:
Oh yeah, that's a leftover :+1: